src/desktop/components/artwork_metadata_stub/templates/contact.jade

```diff
@@ -20,7 +22,8 @@ if artwork.sale && artwork.sale.is_auction
           | #{sa.opening_bid.display}
       else if artwork.sale.is_closed
         | Auction closed
-else if artwork.is_inquireable
+else if artwork.is_inquireable && !(artwork.is_acquireable && auctionPartner)
```

Comment:
This is the byline on the artwork brick.

If the artwork is `inquireable`,  we show "Contact Gallery". Here, we update it to _not_ show that CTA if the artwork is `acquireable` and belongs to an auction partner.